composer.json

```diff
@@ -155,9 +155,11 @@
     "conflict": {
```

Comment:
I don't see any reason to add a conflict. If someone wants to use this on top for whatever reason, why should we care? Conflicts are here to notify people about things that cannot work together. We sure can because we don't care about those bundles anymore.

Reply:
This is for `contao/manager-bundle` only, where it never makes sense to manually require the bundle in my opinion.

Reply:
You're just creating additional rules for Composer to resolve for no reason. There is no conflict so there's no need to add one.

Reply:
Are you sure there is none? We're aliasing a command that potentially already exists

Reply:
As I wrote, we shouldn't alias the command either.

Reply:
Well I think we should, thats why I added it in this PR 😂

Reply:
Then you should probably extract that part into its own PR. Otherwise nothing is going to make it into 4.x.

Reply:
I am with @Toflar on this one. We should not add a conflict and we should not alias the command.